web/src/actions/apd.test.js

```diff
@@ -244,36 +263,34 @@ describe('apd actions', () => {
         });
       });
 
-      it('selects an APD if local storage contains a last APD ID', () => {
-        const apd = {};
+      it('selects an APD if local storage contains a last APD ID', async () => {
```

Comment:
This one's funny because it tests this one unit of a whole flow.  The unit works, the flow does not.

Reply:
😬